def/de/menu_linux.cson

```diff
@@ -141,161 +141,161 @@ Menu:
       "Reflow Selection":
         value: "Reflow Selection"
       Bookmark:
-        value: "Bookmark"
+        value: "Lesezeichen"
         submenu:
           "View All":
-            value: "View All"
+            value: "Alle anzeigen"
           "Toggle Bookmark":
-            value: "Toggle Bookmark"
+            value: "Lesezeichen umschalten"
           "Jump to Next Bookmark":
-            value: "Jump to Next Bookmark"
+            value: "Zum nächsten Lesezeichen springen"
           "Jump to Previous Bookmark":
-            value: "Jump to Previous Bookmark"
+            value: "Zum vorhergehenden Lesezeichen springen"
       "Select Encoding":
-        value: "Select Encoding"
+        value: "Encodierung auswählen"
       "Go to Line":
-        value: "Go to Line"
+        value: "Gehe zu Zeile"
       "Select Grammar":
-        value: "Select Grammar"
+        value: "Grammatik auswählen"
   "&View":
-    value: "View(&V)"
+    value: "Anzeige"
     submenu:
       "Toggle &Full Screen":
-        value: "Toggle Full Screen(&F)"
+        value: "Vollbild ein-/ausschalten"
       "Toggle Menu Bar":
-        value: "Toggle Menu Bar"
+        value: "Menüleiste ein-/ausschalten"
       Panes:
-        value: "Panes"
+        value: "Fensterinhalt"
         submenu:
           "Split Up":
-            value: "Split Up"
+            value: "Nach oben aufteilen"
           "Split Down":
-            value: "Split Down"
+            value: "Nach unten aufteilen"
           "Split Left":
-            value: "Split Left"
+            value: "Nach links aufteilen"
           "Split Right":
-            value: "Split Right"
+            value: "Nach rechts aufteilen"
           "Focus Next Pane":
-            value: "Focus Next Pane"
+            value: "Nächsten Fensterinhalt fokussieren"
           "Focus Previous Pane":
-            value: "Focus Previous Pane"
+            value: "Vorheriger Fensterinhalt fokussieren"
           "Focus Pane Above":
-            value: "Focus Pane Above"
+            value: "Fensterinhalt oberhalb fokussieren"
           "Focus Pane Below":
-            value: "Focus Pane Below"
+            value: "Fensterinhalt unterhalb fokussieren"
           "Focus Pane On Left":
-            value: "Focus Pane On Left"
+            value: "Fensterinhalt links fokussieren"
           "Focus Pane On Right":
-            value: "Focus Pane On Right"
+            value: "Fensterinhalt rechts fokussieren"
           "Close Pane":
-            value: "Close Pane"
+            value: "Gesamten Fensterinhalt schließen"
       Developer:
-        value: "Developer"
+        value: "Entwickler"
         submenu:
           "Open In &Dev Mode…":
-            value: "Open In Dev Mode(&D)..."
+            value: "Öffnen im %Dev Modus..."
```

Comment:
This was mainly because of the hotkey issue... I thought it would be better to stick to the abbreviation "Dev" = Developer = "Entwickler" (german) ... but I think "Dev" is also a known abbreviation in Germany, but I will exchange this in "Öffnen im Entwicklungs-Modus (&D)"